static/app/utils/analytics/workflowAnalyticsEvents.tsx

```diff
@@ -52,6 +52,10 @@ export type TeamInsightsEventParameters = {
     session_id: string;
     wizard_v3: string;
   };
+  'project_creation_page.created': {
+    issue_alert: string;
```

Comment:
@Neo-Zhixing maybe we should have more strict typing here since it looks like there are only 3 values?